api_spec/edge_api.json

```diff
@@ -91,6 +94,70 @@
                     }
                 }
             }
+        },
+        "/bulk-identities": {
+            "post": {
+                "tags": [],
+                "summary": "Bulk insert identities",
+                "description": "Bulk uploading identities and traits.\n\n\n:::info INFO\nIt has a limit of 100 identities per request\n\n:::\n\n \n\n",
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "type": "object",
+                                "example": {
+                                    "data": [
+                                        {
+                                            "$ref": "#/components/examples/PostIdentityExampleData"
+                                        }
+                                    ]
+                                },
+                                "items": {
```

Comment:
This is slightly wrong here. By defining the `"items"` attribute, it thinks we're defining an array, whereas we want an object. I think we just need to remove this `"items"` key and add the `"$ref"` attribute directly at this level instead.

Reply:
This hasn't been resolved. Let's leave all conversations unresolved and I can resolve them once I've confirmed they're resolved.